packages/salesforcedx-apex-replay-debugger/src/core/logContext.ts

```diff
@@ -99,6 +113,88 @@ export class LogContext {
     );
   }
 
+  public getHeapDumps(): ApexHeapDump[] {
+    return this.apexHeapDumps;
+  }
+
+  public scanLogForHeapDumpLines(): boolean {
+    const heapDumpRegex = RegExp(/\|HEAP_DUMP\|/);
+    for (const line of this.logLines) {
+      if (heapDumpRegex.test(line)) {
+        const splitLine = line.split('|');
+        const heapDump = new ApexHeapDump(
+          splitLine[3] /* heapDumpId */,
+          splitLine[4] /* className */,
+          splitLine[5] /* namespace */,
+          Number(splitLine[6]) /* line */
```

Comment:
This is more just as a comment to anyone reading (no changes required). See https://stackoverflow.com/questions/369220/why-should-you-not-use-number-as-a-constructor on why we should not use `new Number(xx)`. What you have with Number(xx) is OK.